test/gyppies/configuration_name_hack/dep/dep.gyp

```diff
@@ -0,0 +1,19 @@
+{
+  "targets": [{
+    "target_name": "copy_dep",
+    "type": "none",
+    "copies": [{
+      "destination": "<(PRODUCT_DIR)",
+      "files": [
+        "dep.c",
+      ],
+    }],
+  }, {
+    "target_name": "dep",
+    "type": "static_library",
+    "dependencies": [ "copy_dep" ],
+    "sources": [
+      "../<(CONFIGURATION_NAME)/dep.c",
```

Comment:
So I'm puttin a file in the output folder in the previous target (`copy_dep` => `dep.c`)